assignments/callbacks.js

```diff
@@ -2,27 +2,38 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 function firstItem(arr, cb) {
   // firstItem passes the first item of the given array to the callback function.
+  cb(arr);
```

Comment:
this is close, but remember its asking for the first item of the array to be passed into the callback.  you are passing the whole array.